packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -41,11 +42,13 @@ export const processConsolidatedDataWebhook = async ({
   status,
   bundle,
   filters,
+  cxPayload,
 }: {
   patient: Pick<Patient, "id" | "cxId">;
   status: ConsolidatedWebhookStatus;
   bundle?: Bundle<Resource>;
   filters?: Filters;
+  cxPayload?: object;
```

Comment:
shouldn't this be required?

Reply:
No because the payload is optional for the customer right?